Source/Core/DolphinWX/GameListCtrl.h

```diff
@@ -109,16 +116,27 @@ class CGameListCtrl : public wxListCtrl
   static bool MultiCompressCB(const std::string& text, float percent, void* arg);
   static bool WiiCompressWarning();
 
-  std::vector<int> m_FlagImageIndex;
-  std::vector<int> m_PlatformImageIndex;
-  std::vector<int> m_EmuStateImageIndex;
-  std::vector<int> m_utility_game_banners;
-  std::vector<std::unique_ptr<GameListItem>> m_ISOFiles;
-
-  int last_column;
-  int last_sort;
-  wxSize lastpos;
-  wxEmuStateTip* toolTip;
+  struct
+  {
+    std::vector<int> flag;
+    std::vector<int> platform;
+    std::vector<int> utility_banner;
+    std::vector<int> emu_state;
+  } m_image_indexes;
+
+  // Actual backing GameListItems are maintained in a background thread and cached to file
+  static constexpr u32 CACHE_REVISION = 0;
```

Comment:
This is supposed to have a comment saying which PR it was last updated in, so that we will get a merge conflict if two PRs increment it to the same value in parallel.